CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+
+When addressing review feedback, it is helpful to the reviewer if additional changes are made in new commits. This allows the reviewer to easily see the delta between what they previously reviewed and the changes you added to address their feedback.
+
+Once a PR has at least two approvals, it can be merged. Here’s how the merge should be handled:
+- If the PR is a single logical commit, the merger should use the “Rebase and merge” option. This keeps the git commit history very clean and simple and eliminates noise from "merge commits."
+- If the PR is more than one logical commit, the merger should use the “Create a merge commit” option.
+- If the PR consists of more than one commit because the author added commits to address feedback, the commits should be squashed into a single commit (or more than one logical commit, if it is a big feature that needs more commits). This can be achieved in one of two ways:
+  - The merger can use the “Squash and merge” option. If they do this, the merger is responsible for cleaning up the commit message according to the previously stated commit message guidance.
+  - The pull request author, after getting the requisite approvals, can reorganize the commits as they see fit (using, for example, git rebase -i) and re-push.
+
+Once the PR is merged, please change the status of the linked issue in [rancher project](https://github.com/rancher/rancher) to "Needs QA". In the following days, the QA team will run tests and process the PR to verify that no regressions were introduced. Please be alert on the issue as the QA team might ask questions related to the PR. 
+
+### Backports
+
+#### RKE code
+
+To understand how backports should be pulled through, RKE project releases and what is included in each release should be described. RKE in general maintains two tracks in parallel which are differentiated by the minor version. Each of these tracks are developped in two different github branches:
+* release/v1.x, (for example [release/v1.5](https://github.com/rancher/rke/tree/release/v1.5)
+* release/v1.(x-1) (for example [release/v1.4](https://github.com/rancher/rke/tree/release/v1.4)
+
+Each track releases at the same time an RKE binary. Each release of RKE officially supports a few Kubernetes versions. Some versions are shared by both tracks but normally the track with the higher minor version includes at least one newer Kubernetes version. For example, as it can be observed in [release v1.4.11](https://github.com/rancher/rke/releases/tag/v1.4.11), the following Kubernetes versions are supported:
+* v1.26.9-rancher1-1
+* v1.25.14-rancher1-1
+* v1.24.17-rancher1-1
+* v1.23.16-rancher2-3
+
+whereas [release v1.5.0](https://github.com/rancher/rke/releases/tag/v1.5.0) includes:
+* v1.25.14-rancher1-1
+* v1.26.9-rancher1-1
+* v1.27.6-rancher1-1
+
+By support, it means we can deploy a cluster with that Kubernetes version using the binray included in that RKE release.
```

Comment:
Suggest: _If a Kubernetes version is **supported**, that means we can deploy a cluster with that Kubernetes version using the binary included in that RKE release._